### DISCUSSION TITLE
Need clarification on SC 1.4.4 AND SC 1.4.10

### DISCUSSION THREAD

**Discussion by AccessibilityEnthusiastic** [w3c/wcag#3383] [Q&A]
Hi Team,

Please share your opinion on below scenario:

When I resize the page to 200 percent, the hamburger button becomes inoperable when using the keyboard. 

However, when the page is in a regular state (100 percent), it works perfectly fine with both the keyboard and mouse interactions. 

The hamburger button does not respond to keyboard input, making it inaccessible to users who rely on keyboard navigation. 
However, it remains operable with a mouse click.

My concern is whether this behavior should be considered a violation or not. 

Is it a must to have a hamburger button accessible by keyboard or not since SC 1.4.4 AND SC 1.4.10 affect people with low vision who rely on a mouse for navigation(not much on a keyboard)

**Comment by awkawk** [comment]
Yes, this is a failure of 2.1.1 (Keyboard) unless the functionality is available via the keyboard in some other way.

**Reply by AccessibilityEnthusiastic** [reply]
@awkawk If the Hamburger button is operable by the keyboard but not with the mouse(in 200 percent or in Relow), what's your opinion in this scenario?

**Reply by awkawk** [reply]
Lots of screen magnification users don't use the mouse and do rely on the keyboard.

**Reply by patrickhlauke** [reply]
> If the Hamburger button is operable by the keyboard but not with the mouse

WCAG currently does start with an assumption that things do work with mouse (and therefore requires an SC to make sure things also work with a keyboard). Unfortunately, there's no SC specifically requiring things work with a mouse (or other pointer), as that is tacitly assumed to be the default...

EDIT: at A/AA at least. I pushed for https://www.w3.org/TR/WCAG21/#concurrent-input-mechanisms to more generally cover scenarios where authors for some reason exclude other input mechanisms ... but that's at AAA

**Comment by JAWS-test** [comment]
WCAG 2.1: [5.2.2 Full pages](https://www.w3.org/TR/WCAG21/#cc2):

> A full page includes each variation of the page that is automatically presented by the page for various screen sizes (e.g. variations in a responsive Web page). Each of these variations needs to conform (or needs to have a conforming alternate version) in order for the entire page to conform.

I.e. even without 200% zoom, the application must be operable with the keyboard at all screen sizes (e.g. mobile device).
